README.md

```diff
@@ -107,26 +130,35 @@ Step 3/8 : RUN apt-get update     && apt-get install -y --no-install-recommends
  ---> Using cache
  ---> 9470f9c9ecea
 Step 4/8 : RUN update-ca-certificates
-
- (snip)
-
-
-Dockerfile    : Dockerfile
-ImageName     : structure
-Registry      :
-Tag           : latest
-CommandResult : CommandResult
+ ---> Using cache
+ ---> 80853c222946
+Step 5/8 : RUN curl -sL https://get.docker.com/ | sh
+ ---> Using cache
+ ---> af17b9b8fb1b
+Step 6/8 : RUN curl -LO https://storage.googleapis.com/container-structure-test/v1.8.0/container-structure-test-linux-amd64     && chmod +x container-structure-test-linux-amd64     && mv container-structure-test-linux-amd64 /usr/local/bin/container-structure-test
+ ---> Using cache
+ ---> bea1dca8e10e
+Step 7/8 : VOLUME /configs
+ ---> Using cache
+ ---> 97e90bf8481b
+Step 8/8 : ENTRYPOINT [ "/usr/local/bin/container-structure-test" ]
+ ---> Using cache
+ ---> 6b9746ab76d8
+Successfully built 6b9746ab76d8
+Successfully tagged structure:latest
+SECURITY WARNING: You are building a Docker image from Windows against a non-Windows Docker host. All files and directories added to build context will have '-rwxr-xr-x' permissions. It is recommended to double check and reset permissions for sensitive files and directories.
 ```
+in which case you will only see the output from Docker, the result object is stored in $result.
 
-Now, we get both the result and the output from the docker command.
-
-In most cases you will want to store the result in a variable for further processing or output to a CI/CD pipeline, like so:
-
+You can verify the existence of the image you just created using `docker images`
 ```
-PS C:\docker> $result = Invoke-DockerBuild . -ImageName structure -PassThru
+PS C:\docker> docker images
 ```
 
-Which would then output the status of the command and store the final result object in `$result`.
+If you want less output, use `-Quiet` switch to output only the final result of the command. Combined with storing the result in a variable, this will give a completely silent execution of the CmdLet.
+
+### Disabling verbose output
+You can also set the enviromenment variable `DOCKER_POSH_QUIET_MODE` to the desired setting for the `-Quiet` switch so you don't have to set it for each invocation of a cmdlet that supports it.
```

Comment:
This should properly be renamed throughout 🤔
```suggestion
You can also set the enviromenment variable `DOCKER_CI_QUIET_MODE` to the desired setting for the `-Quiet` switch so you don't have to set it for each invocation of a cmdlet that supports it.
```

Reply:
True, done.